src/main/java/com/globant/carrito/cart/Carts.java

```diff
@@ -0,0 +1,134 @@
+package com.globant.carrito.cart;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Date;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+import com.globant.carrito.product.Items;
+
+
+@Entity
+public class Carts {
+	
+	@Id
+	@GeneratedValue
+	private int cartId;
+	
+	@ManyToOne(cascade = CascadeType.ALL)
+	private int clientId;
+	
+	@OneToOne(cascade = CascadeType.ALL)
+	private Set<Items> items;
+	
+	@OneToOne(cascade = CascadeType.ALL)

```

Comment:
Lo había pensado que se relaciona con un enum PaymentTypes:

@Entity
public enum PaymentTypes {
    CREDITCARD,
    PAYPAL,
    CASH,
}

No funciona de ese modo?

Date: Wed, 4 Feb 2015 06:55:52 -0800
From: notifications@github.com
To: cart_client@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [cart_client] Avance del shopping cart hasta el 03/02/15. (#1)

In src/main/java/com/globant/carrito/cart/Carts.java:

> +
> +
> +@Entity
> +public class Carts {
> +	
> +	@Id
> +	@GeneratedValue
> +	private int cartId;
> +	
> +	@ManyToOne(cascade = CascadeType.ALL)
> +	private int clientId;
> +	
> +	@OneToOne(cascade = CascadeType.ALL)
> +	private Set<Items> items;
> +	
> +	@OneToOne(cascade = CascadeType.ALL)

Asumo que esto es un número de pago (número de recibo o factura). debería ser @Column en lugar de @OneToOne


—
Reply to this email directly or view it on GitHub.